doc/classes/DisplayServer.xml

```diff
@@ -10,6 +10,13 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="beep" qualifiers="const">
+			<return type="void" />
+			<description>
+				Plays the system beep sound, if possible.
+				[b]Note:[/b] This method is implemented on macOS, Linux (X11/Wayland) and Windows.
```

Comment:
```suggestion
				Plays the system beep sound, if possible. The system beep sound depends on the operating system, but it can be changed or disabled at an OS level by the user.
				Unlike application-level audio playback, the system beep sound will be audible even if the application is muted in the engine's Master bus or in the system audio mixer.
				[b]Note:[/b] This method is implemented on macOS, Linux (X11/Wayland) and Windows.
```

Reply:
I don't think there's much need to be overly specific but this extra addendum is good.
```suggestion
				Plays the beep sound from the operative system, if possible. Because it comes from the OS, the beep sound will be audible even if the application is muted. It may also be disabled for the entire OS by the user.
				[b]Note:[/b] This method is implemented on macOS, Linux (X11/Wayland) and Windows.
```
Or similar.

Reply:
Mickeon's suggestion seems good to amend in before merge @bruvzg

Reply:
Done.